pom.xml

```diff
@@ -388,7 +388,59 @@
             </resource>
         </resources>
         <plugins>
-
+            <!-- Plugin used to extract information on the current git setup.
+                 See: https://github.com/git-commit-id/git-commit-id-maven-plugin -->
+            <plugin>
+              <groupId>io.github.git-commit-id</groupId>
+              <artifactId>git-commit-id-maven-plugin</artifactId>
+              <version>7.0.0</version>
+              <executions>
+                <execution>
+                  <id>get-the-git-infos</id>
+                  <goals>
+                    <goal>revision</goal>
+                  </goals>
+                  <phase>initialize</phase>
+                </execution>
+              </executions>
+              <configuration>
+                <includeOnlyProperties>
+                  <includeOnlyProperty>git.commit.id</includeOnlyProperty>
+                  <includeOnlyProperty>git.branch</includeOnlyProperty>
+                </includeOnlyProperties>
+              </configuration>
+            </plugin>
```

Comment:
Isn't this covered by the `plugin` entry below?

Reply:
Yep! That's a blunder!